### ISSUE TITLE
No information on tracking output response

### ISSUE THREAD

**Issue by nimiew** [buger/goreplay#779]
Had to look at other issues to learn how to track output response. Example given is to track input response... please update the docs.

**Comment by slimus** [comment]
Hi @nimiew ! Did you try `-output-http-track-response` flag? We will add this to our wiki. Thanks!

**Comment by nimiew** [comment]
@slimus yes managed to figure that out. Can you also please include an example using it goreplay with kubernetes? Would be extremely helpful. Thanks!
